### PR TITLE
Update create-onmetal-cloud-servers.md

### PR DESCRIPTION
**Pull request by rsgeinzer**
There have been additional regions in which the on metal servers can be built.  This document only referenced IAD.   DFW and LON have become available.

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-eca42f4b61/how-to/).

File: content/cloud-servers/create-onmetal-cloud-servers.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ up an OnMetal server through the Control Panel.
     name for the server and select the region in which the server
     will reside.
 
-    **Note:** OnMetal servers are currently available only in the
-    Northern Virginia (IAD) region. This list will be expanded as other
+    **Note:** OnMetal servers are available only in the
+    Northern Virginia (IAD), Dallas (DFW) and UK (LON) regions. This list will be expanded as other
```
Comment (RackerKate) [inline content/cloud-servers/create-onmetal-cloud-servers.md]: Suggest adding a serial comma after (DFW), and the text "This list will expand as..."

File: content/cloud-servers/create-onmetal-cloud-servers.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ up an OnMetal server through the Control Panel.
 1.  On the details page for your server, click the link under **Log Into
     Your Server Now** in the right-hand column. For more information,
     see [Connecting to a server using SSH on Linux or Mac OS for further information](/how-to/connecting-to-a-server-using-ssh-on-linux-or-mac-os).
-2.  Use the following command to boot your OnMetal server.
+2.  Use the following command to boot your OnMetal server. (Use the appropriate region for the command).
```
Comment (RackerKate) [inline content/cloud-servers/create-onmetal-cloud-servers.md]: Suggest moving the second period inside of the closing parens.

File: content/cloud-servers/create-onmetal-cloud-servers.md [hunk 3]
```diff
@@ -169,7 +169,7 @@ log in to the server.
 If needed, you can also delete or cancel the server.
 
 1.  Run the following command, replacing the example ID with your
-    server's ID:
+    server's ID: (Use the appropriate region).
```
Comment (RackerKate) [inline content/cloud-servers/create-onmetal-cloud-servers.md]: Suggest moving the second period inside of the closing parens.

**Comment by stephamon** [comment]
@RackerKate and @catlook one of you might want to take over this PR and make the updates yourself. If the OP hasn't done it in the last month, they probably aren't going to do it.

**Comment by RackerKate** [comment]
@stephamon and @catlook I got it.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3a70e004ff/how-to/)

**Pull request merged**